subprojects/core/src/main/java/org/gradle/invocation/DefaultGradle.java

```diff
@@ -94,11 +94,15 @@ public void projectsLoaded(Gradle gradle) {
             }
         });
 
+
```

Comment:
I guess these empty lines can be removed.

Reply:
fixed